src/Authenticator/TokenAuthenticator.php

```diff
@@ -118,4 +119,14 @@ public function authenticate(ServerRequestInterface $request, ResponseInterface
 
         return new Result($user, Result::SUCCESS);
     }
+
+    /**
+     * No-op method.
+     *
+     * @param \Psr\Http\Message\ServerRequestInterface $request A request object.
+     * @return void
+     */
+    public function unauthorizedChallenge(ServerRequestInterface $request)
+    {
```

Comment:
I mistyped, I added `StatelessInterface` not `PersistenceInterface` interface for `TokenAuthenticator` :)

I have only ever seen JWT and custom Token authenticator used in stateless manner so having them implement `StatelessInterface` seems reasonable to me.